catalog/spatial/kml/spatial-kml-util/src/main/java/org/codice/ddf/spatial/kml/converter/MetacardToKml.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.spatial.kml.converter;
+
+import com.vividsolutions.jts.geom.GeometryCollection;
+import com.vividsolutions.jts.geom.LineString;
+import com.vividsolutions.jts.geom.Point;
+import com.vividsolutions.jts.geom.Polygon;
+import com.vividsolutions.jts.io.ParseException;
+import com.vividsolutions.jts.io.WKTReader;
+import ddf.catalog.transform.CatalogTransformerException;
+import de.micromata.opengis.kml.v_2_2_0.Coordinate;
+import de.micromata.opengis.kml.v_2_2_0.Geometry;
+import de.micromata.opengis.kml.v_2_2_0.KmlFactory;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.lang3.StringUtils;
+
+public class MetacardToKml {
+
+  private static final String POINT_TYPE = "Point";
+
+  private static final String LINES_STRING_TYPE = "LineString";
+
+  private static final String POLYGON_TYPE = "Polygon";
+
+  /**
+   * Convert wkt string into JTS Geometry and then from JTS to KML Geo.
+   *
+   * @param wkt
+   * @return
+   * @throws CatalogTransformerException
+   */
+  public static Geometry getKmlGeoFromWkt(final String wkt) throws CatalogTransformerException {
+    if (StringUtils.isBlank(wkt)) {
+      throw new CatalogTransformerException(
+          "WKT was null or empty. Unable to preform KML Transform on Metacard.");
+    }
+
+    com.vividsolutions.jts.geom.Geometry geo = readGeoFromWkt(wkt);
+    Geometry kmlGeo = createKmlGeo(geo);
+    if (!POINT_TYPE.equals(geo.getGeometryType())) {
+      kmlGeo = addPointToKmlGeo(kmlGeo, geo.getCoordinate());
+    }
+    return kmlGeo;
+  }
+
+  private static Geometry createKmlGeo(com.vividsolutions.jts.geom.Geometry geo)
```

Comment:
:pencil2: Rename `geo` to `geometry`

Reply:
I only moved this file, so I'd like to keep it. I think it was specifically kept as Geo (as well as the input parameter as geo instead of geometry) to differentiate between KML Geometry and JTS Geometry

Reply:
Maybe they should be called `kmlGeometry` and `jtsGeometry` then. ;)

Reply:
Fixed.